libs/ledger-live-common/src/bridge/useBridgeTransaction.ts

```diff
@@ -240,7 +242,12 @@ const useBridgeTransaction = <T extends Transaction = Transaction>(
       statusIsPending.current = true; // consider pending until status is resolved (error or success)
 
       Promise.resolve(debounce)
-        .then(() => getAccountBridge(mainAccount, null))
+        .then(() =>
+          isACRE
+            ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+            : getAccountBridge(mainAccount, null),
```

Comment:
I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`